index.html

```diff
@@ -701,35 +701,33 @@ <h4>Visually identical text that is not canonically equivalent</h4>
 					<thead>
 						<tr>
 						   <th>Description</th>
-						   <th colspan=4 style="text-align:center">Examples</th>
-						</tr>
+						   <th style="text-align:center">Examples</th>
+<th style="text-align:center">&nbsp;</th>
+<th style="text-align:center">&nbsp;</th>
+</tr>
 					</thead>
 					<tbody>
 						<tr>
 							<td><strong>Canonically equivalent alternatives</strong><br/>(differences resolved by Unicode Normalization)</td>
-							<td class="exampleChar">&#x625;</td>
-							<td><code class="uname" translate="no">U+0625 ARABIC LETTER ALEF WITH HAMZA BELOW</code></td>
-							<td class="exampleChar">&#x627;&#x655;</td>
-							<td><code class="uname" translate="no">U+0627 ARABIC LETTER ALEF</code> + <code class="uname" translate="no">U+0655 ARABIC HAMZA BELOW</code></td>
-
+							<td class="exampleChar"><img src="https://r12a.github.io/c/Arabic/large/0625.png" alt="&#x065B;" style="height: 4rem; width: 4rem;"></td>
```

Comment:
Let's be sure to move the graphic to our repo and make the link relative?

Reply:
Yes, i didn't do that pending approval for the approach, but we will need to do that before we can publish the doc to TR.